include/RigidBody/Joints.h

```diff
@@ -144,25 +144,43 @@ class BIORBD_API Joints : public RigidBodyDynamics::Model
     /// \param segmentName The name of the segment
     /// \return The biorbd body identification
     ///
-    int GetBodyBiorbdId(
+    int getBodyBiorbdId(
         const utils::String &segmentName) const;
 
     ///
     /// \brief Return the rbdl body identification
     /// \param segmentName The name of the segment
     /// \return The rbdl body identification
     ///
-    int GetBodyRbdlId(
+    int getBodyRbdlId(
         const utils::String &segmentName) const;
 
     ///
     /// \brief Return the Biorbd body identification from rbdl
     /// \param idx The Rbdl body Id
     /// \return The Biorbd body identification
     ///
-    int GetBodyRbdlIdToBiorbdId(
+    int getBodyRbdlIdToBiorbdId(
         const int idx) const;
 
+    ///
+    /// \brief Return the rbdl idx of subtrees of each segments
+    /// \return the rbdl idx of subtrees of each segments
+    ///
+    std::vector<std::vector<unsigned int> > getDofSubTrees();
+
+protected:
+    ///
+    /// \brief Return the rbdl idx of subtrees of each segments
+    /// \param subTrees the rbdl idx of subtrees of each segments to be filled
+    /// \param idx starting index to explore the subtrees
+    /// \return the rbdl idx of subtrees of each segments starting from the specified index
+    ///
+    std::vector<std::vector<unsigned int> > recursiveSubTrees(
```

Comment:
Slight last comment about this function, can you harmonize its name (even though it is protected and therefore less important, I feel that having the name being : getDofSubTreesRecursive, makes it more clear these methods go alongside

Reply:
Done.